WeightProducer/python/MCSampleValues.py

```diff
@@ -34,7 +30,7 @@ class MCSampleHelper():
     __names_to_strip = {
         "generators"    : ["(.pythia)([^_-]*)","(.powheg)([^_-]*)","(.madgraph)([^_-]*)","(.madspin)([^_-]*)","(.amcatnlo)([^_-]*)"],
         "tunes"         : ["(.Tune)([^_]*)"],
-        "added_info"    : ["(.PSweights)([^_]*)"],
+        "added_info"    : ["(.PSweights)([^_]*)","(.PSWeights)([^_]*)"],
```

Comment:
I think the two psweights patterns be combined using `(.PS[W,w]eights)([^_]*)`. I tried it on https://regex101.com/ and it seems to work for the sample lines:
```
stuff_PSweights_stuff
stuff_PSWeights_stuff
```